app/index.js

```diff
@@ -270,94 +278,92 @@ module.exports = yeoman.Base.extend({
       },
       done
     );
-  },
+  }
 
-  copyFiles: function() {
+  writing() {
     this.directory('.', '.');
-  },
+  }
 
-  generateYoRc: function() {
+  // call `this.fs.commit` to finish moving the template files
+  // from memory to destination folder.
+  commit() {
+    var done = this.async();
+    this.fs.commit(done);
```

Comment:
From Yeoman docs:
>It is worth noting that although this.fs exposes commit, you should not call it in your generator. Yeoman calls this internally after the conflicts stage of the run loop.

http://yeoman.io/authoring/file-system.html. It's fine if we use it, but I was wondering whether there are alternatives.

Reply:
@b-admike yep it's a workaround. I eventually calls it because there is no easy better solution for it :(